backend/src/modules/controllers/userController.ts

```diff
@@ -0,0 +1,20 @@
+import { getUserProvider } from 'db/providers/userProvider';
+import { Request, Response } from 'express';
+
+import { TMiddlewareCall } from 'interfaces/middleware/common';
+import { generateInitialDto } from 'utils/dto/dtoUtils';
+import { isError } from 'utils/typeGuards/isError';
+
+const getProfileController = async (req: Request, res: Response, next: TMiddlewareCall) => {
+  try {
+    const { id } = res.locals;
+    const profileInfo = await getUserProvider(id);
+    res.json(generateInitialDto(profileInfo));
+  } catch (error) {
+    if (isError(error)) {
+      next(error);
+    }
+  }

```

Comment:
Let's create inside modules new folder `users/controllers `
And `index.ts` file that should export `getUser.ts` file inside the same folder. For each method we will define separate file.
Rename please `getProfileController` to the `getProfileInformation`
Also, try to get an `id` from `req.query` or query string params
Rename please `getUserProvider` to the `getUser`
No need to call next inside catch
Filter please all the imports
Rename please `dtoUtil`s to the `utils`
Rename please `db/providers/userProvider` to the `db/providers/users/index.ts` and define all the methods inside this file
Send please response and return it